docs/user/openstack/etcd-ephemeral-disk.md

```diff
@@ -0,0 +1,203 @@
+# Moving etcd to an ephemeral local disk
+
+You can move etcd from a root volume (Cinder) to a dedicated ephemeral local disk to prevent or resolve performance issues.
+
+## Prerequisites
+
+* An OpenStack cloud with Cinder being functional and enough available storage (at least 75 GB) to accommodate 3 Root Volumes for the OpenShift control plane.
+* An OpenStack cloud whith Nova not using Ceph as a backend for ephemeral disks.
+* OpenShift will be deployed with IPI for now; UPI is not yet documented but technically possible.
+* The MachineConfigPool must match metadata.labels[machineconfiguration.openshift.io/role]. This applies to a controller, worker, or a custom pool.
+* The control-plane machine’s auxiliary storage device, such as /dev/vdb, must match the vdb. Change this reference in all places in the file.
+
+## Procedure
+
+* Create a Nova flavor for the Control Plane which allows 10 GiB of Ephemeral Disk:
+
+```bash
+openstack flavor create --ram 16384 --disk 25 --ephemeral 10 --vcpus 4 m1.xlarge
```

Comment:
That is probably not enough disk space. We recommend at least 100GB disk space for the control plane nodes: https://docs.openshift.com/container-platform/4.14/installing/installing_openstack/installing-openstack-installer-custom.html#installation-osp-control-machines_installing-openstack-installer-custom

Also, the recommended memory and CPU depends on the expected cluster size: https://docs.openshift.com/container-platform/4.14/scalability_and_performance/recommended-performance-scale-practices/recommended-control-plane-practices.html#master-node-sizing_recommended-control-plane-practices

Reply:
I'll remove the RAM, disk and CPU numbers, the only thing of value here is the ephemeral flag.